esx_service/vmci/vmci_server.c

```diff
@@ -154,6 +154,7 @@ vmci_get_one_op(const int s,    // socket to listen on
                ret, strerror(errno), b, MAGIC);
       close(client_socket);
       errno = saved_errno;
+      CHECK_ERRNO(ret);
```

Comment:
it's not  redundant since prior line *recovers* erno to the one before "close". If returned value is not MAGIC, errno "before close" would have been 0 and we need to set it.